test/tests/wallet/Test_PegOut.cpp

```diff
@@ -36,16 +37,17 @@ TEST_CASE("Wallet - Peg-out")
     REQUIRE(pegouts[0].address == "tltc1qh50sy0823vxn4l9zk2820w4fuj0q4fgjza5vv");
     REQUIRE(pegouts[0].amount == 3'000'000);
 
+    uint64_t expected_fee = 10'000 * Weight::Calculate({ .num_kernels = 1, .num_owner_sigs = 1, .num_outputs = 2 });
```

Comment:
Maybe make fee_rate a const var and also use in CreatePegOutTx.

Reply:
It's not meant to be a const. It's affected by the fee field in the gui.

Reply:
I meant just in the test. Because right now there's 10'000 in two places.